### PR TITLE
Book store develop

### PR DESCRIPTION
**Pull request by StoneBlood-bit**

### CODE REVIEW

File: src/main/java/mate/academy/BookStoreApplication.java [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    public static void main(String[] args) {
+        SpringApplication.run(BookStoreApplication.class, args);
+    }
 
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return new CommandLineRunner() {
+            @Override
+            public void run(String... args) throws Exception {
+            }
```
Comment (liliia-ponomarenko) [inline src/main/java/mate/academy/BookStoreApplication.java]: Let’s  create a book for testing your application

File: src/main/java/mate/academy/repository/BookRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    private final SessionFactory sessionFactory;
+
+    @Autowired
```
Comment (liliia-ponomarenko) [inline src/main/java/mate/academy/repository/BookRepositoryImpl.java]: Let’s  use RequiredArgsConstructor (fix it everywhere)

File: src/main/java/mate/academy/repository/BookRepositoryImpl.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            }
+        }
+    }
+
+    @Override
+    public List<Book> findAll() {
+        Session session = null;
+        try {
+            session = sessionFactory.openSession();
+            return session.createQuery("from Book", Book.class).list();
```
Comment (liliia-ponomarenko) [inline src/main/java/mate/academy/repository/BookRepositoryImpl.java]: you can use try with resources.

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! A few comments ;)

**Review by ahoienko — APPROVED** [review]
Good job!

**Pull request merged**